mlir/lib/Dialect/Tosa/IR/TosaCanonicalizations.cpp

```diff
@@ -73,41 +73,54 @@ struct SqrtReciprocalOptimization : public OpRewritePattern<tosa::PowOp> {
                                 PatternRewriter &rewriter) const override {
     // Check that the PowOp has a single user
     if (!op->hasOneUse())
-      return rewriter.notifyMatchFailure(op, "pow operator has more than one user");
```

Comment:
Can you please not reformat this file? It will make merging upstream harder

Reply:
I think it was @ehsan-toosi's vscode configuration who did 😅  I'll try to revert this.